src/sentry/api/endpoints/event_attachment_details.py

```diff
@@ -48,6 +49,7 @@ def has_object_permission(self, request: Request, view, project):
 
 @region_silo_endpoint
 class EventAttachmentDetailsEndpoint(ProjectEndpoint):
+    owner = ApiOwner.OWNERS_INGEST
```

Comment:
Unless you introduce another option for "processing" team, I guess ingest would be the most appropriate

Reply:
TBH, I have no idea what the difference is. @ashwoods

Reply:
For Ingest we got rid of the separate owners team. I'll look into this for processing as well.
Please just use team-processing.